app/viewmodels/checkAnswers/AnswerSummary.scala

```diff
@@ -22,5 +22,6 @@ import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 
 trait AnswerSummary {
-  def row(answers: UserAnswers, taxYear: TaxYear, businessId: BusinessId, userType: UserType)(implicit messages: Messages): Option[SummaryListRow]
+  def row(answers: UserAnswers, taxYear: TaxYear, businessId: BusinessId, userType: UserType, rightTextAlign: Boolean = true)(implicit
```

Comment:
Summary rows need the option to have rightTextAlign or leftTextAlign. See Confluence Team C's WaysOfWorking for details